tdrs-backend/tdpservice/settings/common.py

```diff
@@ -42,6 +42,7 @@ class Common(Configuration):
         "django.contrib.messages.middleware.MessageMiddleware",
         "django.middleware.clickjacking.XFrameOptionsMiddleware",
         "corsheaders.middleware.CorsMiddleware",
+        "tdpservice.users.api.middleware.AuthUpdateMiddleware"
```

Comment:
Using middleware to update the session so that it happens on every request (without having to add code to every request)